scripts/compare.py

```diff
@@ -1,21 +1,36 @@
 #!/usr/bin/env python3
 
 import sys
+import re
 
 logfile_1 = sys.argv[1]
 logfile_2 = sys.argv[2]
-mintime   = 0.01
-# minchange = 0.035
-minchange = 0.035
-  
+if len(sys.argv) > 3:
+    mintime = float(sys.argv[3])
+else:
+    mintime = 0.01
+if len(sys.argv) > 4:
+    minchange = float(sys.argv[4])
+else:
+    minchange = 0.035
+
+testname = re.compile(r'^src/.*\[(?P<name>.*)\]$')
+
+def readName(input):
+    r = testname.match(input)
+    if r is None:
+        return input
+    else:
+        return r.group("name")
```

Comment:
I think the names will be unambiguous so we should not print the path to the python runner.